celery/utils/time.py

```diff
@@ -383,3 +385,20 @@ def utcoffset(time=_time, localtime=_time.localtime):
 def adjust_timestamp(ts, offset, here=utcoffset):
     """Adjust timestamp based on provided utcoffset."""
     return ts - (offset - here()) * 3600
+
+
+def get_exponential_backoff_interval(
+    factor,
+    retries,
+    maximum,
+    full_jitter=False
```

Comment:
Yes, the documentation should be updated, I didn't make any changes until we finalized on the structure and settings. If you agree with the changes then we should somehow transfer these to your PR. Please tell me how you would like to proceed.